packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -0,0 +1,44 @@
+import { GraphbackOperationType} from '../api';
```

Comment:
@craicoverflow This is the only change in this PR. I have added one place for mapping every CRUD element. This needs to be extended and used in plugins